src/js/step.js

```diff
@@ -328,10 +288,9 @@ export class Step extends Evented {
       this.target.classList.remove('shepherd-enabled', 'shepherd-target');
     }
 
-    if (this.popper) {
-      this.popper.destroy();
+    if (this.tooltipElem) {
+      this.tooltipElem.hide();
```

Comment:
So tippy will destroy all its DOM when we cancel the tour without us manually destroying here?

Reply:
... but then again, it looks like that's part of `Tour.done`, too. A few lines later we do:

```js
    if (Shepherd.activeTour) {
      Shepherd.activeTour.steps.forEach((step) => {
        step.destroy();
      });
    }
```

Running through the tour in the demo, all tippy elements and DOM changes do, indeed, get cleaned up.

Reply:
The idea is that `done` calls all that, yeah.

Reply:
Shepherd did not use to cleanup after itself, so I just want to make sure things are cleaned up. If they are, we're good.

Reply:
May just want to move things back to `done`. The less moving and changing the better 😄

Reply:
I'm not sure I'm following... are you saying this can this be resolved? I think `hide` and `done` are both doing what they're supposed to now.

Reply:
I'm not sure what I was saying here anymore 😂. We can resolve for now.